docs/src/extend/shareable-configs.md

```diff
@@ -66,59 +72,62 @@ Be sure to replace `eslint-config-myconfig` with the actual name of your module.
 
 ## Using a Shareable Config
 
-Shareable configs are designed to work with the `extends` feature of `.eslintrc` files. Instead of using a file path for the value of `extends`, use your module name. For example:
+To use a shareable config, include it in the `extends` field of an `.eslintrc` file. For the value, use your module name. For example:
```

Comment:
I'd keep the "Shareable configs are designed to work with the `extends` feature of `.eslintrc` files." sentence so that it isn't expected that they can be used in other ways. For example, we had a bug report that shareable configs don't work with `--config` (https://github.com/eslint/eslint/issues/15697).

Reply:
i think the proposed changes are pretty clear that the shareable config should be in the ESLint config file. 

Do you think we should add some sentence like: "You cannot use shareable configs with the ESLint CLI `--config` flag"?

Reply:
> Do you think we should add some sentence like: "You cannot use shareable configs with the ESLint CLI `--config` flag"?

Yes, I think that would be good.

(although, technically it is possible to use a shareable config package with `--config`, e.g. `eslint --config node_modules/eslint-config-airbnb/index.js`)

Reply:
I think there's still some ambiguity with the current wording. "Include it in the extends field". What is "it"? (We know it's a string, but maybe "it" should "the config name" or something more specific?)